packages/vitest/src/node/core.ts

```diff
@@ -94,6 +95,7 @@ export class Vitest {
   /** @internal */ reporters: Reporter[] = undefined!
   /** @internal */ vitenode: ViteNodeServer = undefined!
   /** @internal */ runner: ViteNodeRunner = undefined!
+  /** @internal */ testRun: TestRun = undefined!
```

Comment:
We should use the `_*` convention for public internal fields, the old ones are kept as is for backwards compatibility